packages/x-components/tests/e2e/cucumber/keyboard-navigation.spec.ts

```diff
@@ -11,7 +11,9 @@ type Direction = 'above' | 'below' | 'on the left' | 'on the right';
 
 // Scenario 1
 And('{string} element position is stored', (focusableElement: string) => {
-  cy.getByDataTest(focusableElement).focus().as('originalElement');
+  return focusableElement === 'search-input'
```

Comment:
When the `focusableElement` is the `search-box` and there is only one, the `last()` is the `first` one too :trollface: , so you can remove the if statement:

```cucumber
// Scenario 1
And('{string} element position is stored', (focusableElement: string) => {
  return cy.getByDataTest(focusableElement).last().focus().as('originalElement');
});
```

Reply:
Hahaha that's what I had at first and I changed it because I thought you were going to complain